### ISSUE TITLE
Please rememember to commit

### ISSUE THREAD

**Issue by mikewilzn** [EHanz/PackageStack#6]
I'm noticing we're already straying away from the idea of versioning. It becomes really hard to follow when your commits contain more than one line of code. Please try to keep proper commit records and make the commit messages as detailed as possible. I find it helps to past the code you're committing into the commit message so that everyone knows what you're changing.

**Comment by jkcarney** [comment]
ThankS!
almost forgot

**Comment by astruxie** [comment]
Good idea! i personally always comment where i am changing my code that way it is right there in the source code
